### PR TITLE
add new design for summary screen

### PR DESCRIPTION
**Pull request by julia-andersson**
## [APP-XXX]

- new design for summary screen

## Checklist

- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift [hunk 1]
```diff
@@ -10,114 +11,137 @@ public struct SubmitClaimSummaryScreen: View {
...
         PresentableStoreLens(
             SubmitClaimStore.self,
             getter: { state in
                 state.summaryStep
             }
         ) { summaryStep in
-            hText(summaryStep?.title ?? "", style: .title3)
-                .padding(.top, UIScreen.main.bounds.size.height / 5)
-                .foregroundColor(hLabelColor.secondary)
+            createRow(with: L10n.claimsCase, and: summaryStep?.title ?? "")
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift:59]: I think this should be "Broken phone or whatever was chosen from claimsTriaging

**Review by sladan-hedvig — CHANGES_REQUESTED** [review]

File: Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift (hunk 1 above)
Comment (julia-andersson) [reply Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift:59]: @sladan-hedvig I don't think we are getting that attribute from backend

Comment (sladan-hedvig) [reply Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift:59]: We can save it when we are starting the flow and show it using that.

Comment (julia-andersson) [reply Projects/Claims/Sources/Views/SubmitClaim/SummaryInfo/SubmitClaimSummaryScreen.swift:65]: title is now "broken item"

**Comment by julia-andersson** [comment]
> 

Removed info button

**Pull request merged**